cores/esp8266/Updater.h

```diff
@@ -141,18 +148,35 @@ class UpdaterClass {
       return written;
     }
 
+#ifdef VERIFY_SIGNATURE
+    int addCA(const uint8_t *cert, int *len);
+#endif
+
   private:
     void _reset();
     bool _writeBuffer();
 
     bool _verifyHeader(uint8_t data);
     bool _verifyEnd();
 
+#ifdef VERIFY_SIGNATURE
+    CA_CERT_CTX *_ca_ctx;
```

Comment:
Ok, i see now why the headers were needed. If you go for the approach with `VerificationTraits` interface (call it some other way if you like), you will not have to expose these internals to the users, introducing axTLS names into the sketch global namespace.